tests/test_post_processor.py

```diff
@@ -1,45 +1,51 @@
 import pytest
+
 from omigami.post_processor import PostProcessor
-from omigami.outer_loop import OuterLoopResults
 from omigami.feature_evaluator import FeatureEvaluationResults
-from omigami.models import FeatureRanks, ScoreCurve
+from omigami.data import FeatureRanks, ScoreCurve, OuterLoopResults
 
 
 @pytest.fixture
 def outer_loop_results():
```

Comment:
we could probably improve these fixtures moving it to conftest and sharing some code with the results fixture